platforms/documentation/docs/src/docs/userguide/reference/best-practices/best_practices_general.adoc

```diff
@@ -142,6 +142,229 @@ include::sample[dir="snippets/bestPractices/usePluginsBlock-do/groovy",files="bu
 
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
 
+[[dont_assume_plugin_order]]
+== Don’t Assume your Plugin is Applied after Another
+
+Gradle's plugin application is deterministic, but the effective execution order is complex and not guaranteed to follow a visible sequence, especially across multiple build scripts, projects, convention plugins, or included builds.
+
+As a result, you should not write build logic or plugins that depend on a specific plugin application order.
+
+=== Explanation
+
+In a single `build.gradle(.kts)` file, plugins appear to be applied sequentially:
+
+[source,kotlin]
+.build.gradle.kts
+----
+plugins {
+    id("pluginA")
+    id("pluginB")
+}
+----
+
+However, in <<multi_project_builds.adoc#multi_project_builds,multi-project builds>>, with multiple `build.gradle(.kts)` files or a <<plugins.adoc#sec:convention_plugins,convention plugin>>, plugin application can be hard to determine:
+
+[source,kotlin]
+.project.gradle.kts
```

Comment:
What is this project.gradle.kts? 

How does it apply for the single build.gradle.kts shown above?

Reply:
I'm also not sure how this is meant to relate to the build file above.  This needs more clarity.

Reply:
more copy pasta. fixed.